lib/paperclip.rb

```diff
@@ -56,6 +56,7 @@
 require 'paperclip/attachment_registry'
 require 'paperclip/filename_cleaner'
 require 'paperclip/rails_environment'
+require 'paperclip/deprecations'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.